.editorconfig

```diff
@@ -8,6 +11,4 @@ trim_trailing_whitespace = true
 insert_final_newline = true
 indent_style = space
 indent_size = 4
-
-[*.{yml,yaml}]
```

Comment:
what's the reason for changing this?

Reply:
same as everywhere else

Reply:
what's everywhere?

Reply:
Why does formatting matter so much :smile: ?

https://github.com/alloy-rs/core/tree/main/.github/workflows
https://github.com/gakonst/ethers-rs/tree/master/.github/workflows
https://github.com/foundry-rs/foundry/tree/master/.github/workflows